src/pages/components/breadcrumbs.mdx

```diff
@@ -10,17 +10,18 @@ components:
 <Usage>
   <Use>
     <ul>
-      <li>To inform the user of where they are in a nested navigation</li>
-      <li>To quickly navigate back to a parent page</li>
+      <li>To show the user where they are in a nested navigation</li>
+      <li>To provide a quick way to navigate to ancestor pages</li>
     </ul>
   </Use>
 </Usage>
 
-## Examples
+## How to use it
 
 ### Default
 
-Breadcrumbs work in combination with the [Anchor](/components/anchor) component for linking.
+Breadcrumbs and the [Anchor](/components/anchor) component work together. Each element (a "crumb" if
```

Comment:
I want to raise the issue of curly quotes again. What is the consensus? I think it's helpful to note that brand uses balanced quotes. See https://brandland.zendesk.com/brand-identity#relationshapes.

Reply:
I think since I'm not even sure how to type a curly quote intentionally, my vote would be to go with straight quotes so we don't have to rely on a linter.

Reply:
I don't have a strong opinion on this. I feel curly quotes better match our website and make the Garden website cohere more with our brand.

That said, it does seem technically challenging (at least more than straight quotes) so I'd leave the final decision with those who would maintain the system.

Reply:
Good point. Looks like zendesk.com is [sporting curly](https://share.getcloudapp.com/ApuAL9b8) and so is [brandland](https://share.getcloudapp.com/rRulob42).

Reply:
@allisonacs What do you think?

Reply:
Again, I want to emphasize that we can make this trivial by adding an automated linter to the build process. No need to stress about typing `Alt + [` etc.

Reply:
Let's please use smart/balanced/curly quotes, especially since we can make it happen via linter. Couple reasons:
- We're dealing with code, where straight quotes are important syntax. So it's important to clearly demonstrate that we're being intentional
- It's proper typography, we are a proper design system
- Consistent with brandland/zendesk.com
- Aesthetics

Reply:
"Balanced quotes" should be occurring in our remark build process. Anything in markdown should get this automatically! https://www.gatsbyjs.org/packages/gatsby-remark-smartypants/

Reply:
Amazing. Also fixes curly apostrophes. I wasn't looking close enough at the deployment. Nice one, @austin94.